boxcli/init.go

```diff
@@ -9,19 +9,29 @@ import (
 	"go.jetpack.io/devbox"
 )
 
+type initCmdFlags struct {
```

Comment:
So ... `init` is the one command I was still imagining would work as `devbox init my/project` instead of `devbox init -c my/project`

Thoughts on whether we should keep init working as before?

Reply:
I don't feel strongly, I can change it back ....

Reply:
How do you feel about keeping it for consistency?